x/stake/msg.go

```diff
@@ -9,7 +9,9 @@ import (
 )
 
 // name to idetify transaction types
-var MsgType = "stake"
+const MsgType = "stake"
+
+const StakingToken = "fermion"
```

Comment:
turn on gometalinter - this generates a warning

Reply:
I don't think we should have StakingToken as a code constant - think it makes more sense belonging with the Params so we can initialize at genesis - to allow for the same tests we should should make the `ValidateBasic()` function a return from an initializable function `ValidateBasic(bondDenom string)` function